### PR TITLE
ss-285: add images storing service

### PR DESCRIPTION
**Pull request by Evcat6**

### CODE REVIEW

File: backend/src/migrations/20230405072520_add_images.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    AVATAR_URL: 'avatarUrl',
+    CREATED_AT: 'createdAt',
+    UPDATED_AT: 'updatedAt',
+};
+const RelationRule = {
+    CASCADE: 'CASCADE',
+    SET_NULL: 'SET NULL',
+};
+async function up(knex: Knex): Promise<void> {
+    await knex.schema.dropTableIfExists(TABLE_NAME);
```
Comment (anton-otroshchenko) [inline backend/src/migrations/20230405072520_add_images.ts:31]: You don't need to drop table, you can edit the existing one instead

**Review by anton-otroshchenko — CHANGES_REQUESTED** [review]

File: backend/src/migrations/20230405072520_add_images.ts (hunk 1 above)
Comment (Evcat6) [reply backend/src/migrations/20230405072520_add_images.ts:10]: Resolved in [6ccc578](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/314/commits/6ccc578e7713a42dbb64accaa2bbbecc31440000)

File: backend/package.json [hunk 2]
```diff
@@ -33,10 +33,12 @@
         "@fastify/cors": "8.2.1",
         "@fastify/swagger": "8.3.1",
         "@fastify/swagger-ui": "1.4.0",
+        "axios": "1.3.4",
```
Comment (nikita-remeslov) [inline backend/package.json:36]: I think we can do it without axios

File: shared/src/enums/exception-message.enum.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ enum ExceptionMessage {
     INVALID_CREDENTIALS = 'You entered an incorrect e-mail address or password',
     EMAIL_ALREADY_EXISTS = 'User with this email already exists',
     INVALID_TOKEN = 'Invalid token',
-    AUTHORIZE_ERROR = 'authorize error'
+    AUTHORIZE_ERROR = 'authorize error',
```
Comment (nikita-remeslov) [inline shared/src/enums/exception-message.enum.ts:5]:
```suggestion
    AUTHORIZE_ERROR = 'You should authorize',
```

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: backend/package.json (hunk 2 above)
Comment (Evcat6) [reply backend/package.json]: Resolved in [2253ca5](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/314/commits/2253ca5bf25402433bc32c8786d726d6d4602dd3)

File: shared/src/enums/exception-message.enum.ts (hunk 3 above)
Comment (Evcat6) [reply shared/src/enums/exception-message.enum.ts]: Resolved in [fe2799b](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/314/commits/fe2799be923f4707b833676e199dd3c160b9de92)

**Review by teslmik — DISMISSED** [review]

**Comment by VItaliy-Kalenichenko** [comment]
Resolve conflicts

File: backend/src/migrations/20230405072520_add_images.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'user_profile';
+
+const ColumnName = {
+    AVATAR_URL: 'avatarUrl',
+};
+
+async function up(knex: Knex): Promise<void> {
+    await knex.schema.dropTableIfExists(TABLE_NAME);
```
Comment (VItaliy-Kalenichenko) [inline backend/src/migrations/20230405072520_add_images.ts:10]: Why are you going to drop this table?

Comment (Evcat6) [reply backend/src/migrations/20230405072520_add_images.ts:10]: Resolved in [6ccc578](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/314/commits/6ccc578e7713a42dbb64accaa2bbbecc31440000)

**Review by anton-otroshchenko — APPROVED** [review]

**Review by Holovashcenko — APPROVED** [review]

**Pull request merged**